[OSDOCS-10299]Update/Remove Cluster Upgrade Titles (#75119)
openshift/openshift-docs: upgrading/rosa-upgrading-sts.adoc

Upgrades are manually initiated or automatically scheduled. Red Hat Site Reliability Engineers (SREs) monitor upgrade progress and remedy any issues encountered.

[id="rosa-sts-upgrading-a-cluster-with-sts"]
== Upgrading a ROSA Classic cluster

There are two methods to upgrade Classic {product-title} (ROSA) clusters:

* Individual upgrades through the ROSA CLI (`rosa`)
* Individual upgrades through the {cluster-manager} console

[NOTE]
====

> When following a scheduled upgrade policy, there might be a delay of an hour or more before the upgrade process begins, even if it is an immediate upgrade. Additionally, the duration of the upgrade might vary based on your workload configuration.

Reviewer: ```suggestion
When you follow a scheduled upgrade policy, a delay of an hour or more before the upgrade process begins might occur, even if the upgrade is configured to begin immediately. Additionally, the duration of the upgrade might vary based on your workload configuration.
```
- Consider editing to avoid "there is" and "it is" constructions, [per the ISG](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#t). Generally, those types of constructions lead to wordiness and weaken a sentence's impact by getting through the subject/verb combination without really saying anything.
